Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.0 → 1.7.0

[_[change-log](https://github.com/svenfuchs/i18n/releases), [source-code](https://github.com/svenfuchs/i18n)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.6.0...v1.7.0).

These are the first 10 commits:
- (38a323e) [I18n::Backend::Chain#translations merge from all backends](https://github.com/ruby-i18n/i18n/commit/38a323e89bd5fcfc0b673d5069d850a22f61e013)
- (daa63b4) [Update spec for new behavior of chain backend](https://github.com/ruby-i18n/i18n/commit/daa63b4cc531f61d9fb1da6f40bf89801d0ef6ac)
- (d298938) [Use each_with_object and more descriptive names](https://github.com/ruby-i18n/i18n/commit/d2989389e3fdd9194cd1d3d3b78988f14cda23c9)
- (b7e5195) [Merge branch 'master' into patch-1](https://github.com/ruby-i18n/i18n/commit/b7e519507fed8cb2cff8968fd52f13a482195a21)
- (7bfc725) [Merge pull request #470 from gburgett/patch-1](https://github.com/ruby-i18n/i18n/commit/7bfc72568f296dc73431059cd3bb2aa9b7112bbf)
- (74a03cc) [Raise disabled during boot inside fallback](https://github.com/ruby-i18n/i18n/commit/74a03cc1412f887fafcf62739734ab434979c09b)
- (ba83938) [Merge pull request #475 from KaanOzkan/raise-disabled](https://github.com/ruby-i18n/i18n/commit/ba83938beb6f9a042ac450c9fd3253de0c9fb6b1)
- (69bd4a7) [Fix typo :)](https://github.com/ruby-i18n/i18n/commit/69bd4a75a89acbf35589c919917a98f6b34098d2)
- (0c5dab4) [Merge pull request #476 from TaigaMikami/master](https://github.com/ruby-i18n/i18n/commit/0c5dab494d9b043e00662d8e789229c33045c024)
- (e49d862) [#include is public since ruby 2.1](https://github.com/ruby-i18n/i18n/commit/e49d8624851c49a1129d63fd15776a5995617245)


</details>